callouts/python/extproc/example/ext_proc_client.py

```diff
@@ -28,18 +28,31 @@
 )
 
 
+def _make_channel(
+  address: tuple[str, int], key: str | None = None
+) -> grpc.Channel:
+  addr_str = _addr_to_str(address)
+  if key:
+    with open(key, 'rb') as file:
+      creds = grpc.ssl_channel_credentials(file.read())
+      return grpc.secure_channel(addr_str, creds)
+  else:
+    return grpc.insecure_channel(addr_str)
+
+
 def make_json_request(
-  json_list: list[str], address: tuple[str, int]
+  json_list: list[str], address: tuple[str, int], key: str | None = None
 ) -> Iterator[ProcessingResponse]:
   """Make requests to a callout service with ProcessingRequest json data.
   Args:
     json_list: A list of json strings representing ProcessingRequest data.
     address: ip address of the callout server.
+    key: Local filepath to a chain authenication certificate.
```

Comment:
Lets not use the Cert chain as a way of saying tls or not. we configure the client to accept unverified certs. Cert chain is overkill. We have a boolean (-plaintext) that allows us to pick plaintext or tls.

Reply:
I can make a follow-up pr to address this, technically they can provide either the cert chain or root cert.